packages/neon/neon_talk/lib/src/pages/room.dart

```diff
@@ -121,14 +129,44 @@ class _TalkRoomPageState extends State<TalkRoomPage> {
                   final previousMessage =
                       messagesResult.requireData.length > index + 1 ? messagesResult.requireData[index + 1] : null;
 
+                  Widget child = TalkMessage(
+                    chatMessage: message,
+                    lastCommonRead: lastCommonReadSnapshot.data,
+                    previousChatMessage: previousMessage,
+                  );
+
+                  if (previousMessage != null) {
```

Comment:
imo the date information is also relevant for the first message in a chat. If there is no previous message just show the date.